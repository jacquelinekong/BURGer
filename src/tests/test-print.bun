print("Hello World!");
print("5+3");
print("!@#$%^&*()-=_+,./<>?~``[]{}|:;\t");
print(5*6);
print("\n");
