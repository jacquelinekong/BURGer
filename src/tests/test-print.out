Hello World!5+3!@#$%^&*()-=_+,./<>?~``[]{}|:;	30
